recipes/default.rb

```diff
@@ -18,7 +18,7 @@
 #
 
 # jenkins 1.612 and above requires java 7
-node.default['java']['jdk_version'] = '7'
+node.default['java']['jdk_version'] = '8'
```

Comment:
I think you need to rebase this branch again since this change shouldn't be here.